packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -1195,6 +1199,8 @@ Okta defines a number of reserved scopes and claims that can't be overridden.
 
 `openid`, `profile`, `email`, `address`, `phone`, `offline_access`, and `groups` are available to ID tokens and access tokens, using either the Okta Org Authorization Server or a Custom Authorization Server. For details, see [Scopes](#access-token-scopes-and-claims). All of these scopes except `groups` are defined in the OpenID Connect specification.
 
+Additionally, we have reserved the scope `online_access`. <ApiLifecycle access="ea" /> as it has a particular meaning similar to `offline_access`.
```

Comment:
Is it `similar to offline_access`?

Reply:
Perhaps the phrasing wasn't great, what I mean is that offline_access has a particular meaning as well. Not that the two scopes have a similar meanings.

Reply:
Since this PR came out for review, the Native SSO API reference was merged, and I found a place to include this information with the new addition of the device_sso scope.